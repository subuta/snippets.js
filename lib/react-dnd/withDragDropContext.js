import { build, format, snippets as s } from 'bld.js'

export default () => {
  const imports = s.import([
    ['react-dnd', null, [
      ['DragDropContext']
    ]],
    ['react-dnd-html5-backend', 'HTML5Backend']
  ])

  return build`
    ${imports}
    
    ${s.export('DragDropContext(HTML5Backend)')}
  `
}
