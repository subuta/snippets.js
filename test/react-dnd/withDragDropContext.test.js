/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import withDragDropContext from 'lib/react-dnd/withDragDropContext'

describe('react-dnd/withDragDropContext', () => {
  it('should create withDragDropContext for react-dnd', () => {
    expect(format(withDragDropContext())).toMatchSnapshot()
  })
})
