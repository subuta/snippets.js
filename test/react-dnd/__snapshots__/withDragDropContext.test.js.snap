// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-dnd/withDragDropContext should create withDragDropContext for react-dnd 1`] = `
"import {DragDropContext} from 'react-dnd'
import HTML5Backend from 'react-dnd-html5-backend'

export default DragDropContext(HTML5Backend)
"
`;
